paper-menu.html

```diff
@@ -27,34 +27,39 @@
       background: var(--paper-menu-background-color, --primary-background-color);
       color: var(--paper-menu-color, --primary-text-color);
 
-      mixin(--paper-menu);
+      @apply(--paper-menu);
     }
 
     :host > ::content > [disabled] {
       color: var(--paper-menu-disabled-color, --disabled-text-color);
     }
 
-    :host > ::content > .iron-selected {
+    :host > ::content > *:focus {
       position: relative;
+      outline: 0;
     }
 
-    :host > ::content > .iron-selected::after {
+    :host > ::content > *:focus:after {
       @apply(--layout-fit);
-
       background: currentColor;
       /* FIXME move to paper-styles for next widget */
```

Comment:
How about
```css
opacity: var(--paper-menu-focused-item-opacity-greyscale, --paper-focused-opacity-greyscale);`
```
and
```css
opacity: var(--paper-menu-focused-item-opacity-colored, --paper-focused-opacity-colored);`
```
where `--paper-focused-opacity-greyscale` and `--paper-focused-opacity-colored` are defined by something like `common.html` in `paper-styles`?